### DISCUSSION TITLE
Pending applications bug?

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#404] [General] [answered]
It appears that pending applications are visible in your profile under both "pending applications" and "current memberships". I would expect it to move from the former to the latter once an application has been approved.

![Screenshot 2021-08-10 12 16 59](https://user-images.githubusercontent.com/87815729/128850459-aab8b5ff-479a-4191-9362-037847d33894.png)
![Screenshot 2021-08-10 12 16 45](https://user-images.githubusercontent.com/87815729/128850464-3c6921dd-cba5-4406-848e-0b747ad8df35.png)

**Comment by Comoque1** [comment, marked as answer]
Hi Flori, appreciate your feedback!!
"Pending applications" is cleaned, after the browser is refreshed. Still, if an user is added from an admin to community, the "Apply" button is available to the user for the corresponding entity (i.e. Ecoverse). I'll log this issue in the backlog.

**Reply by Comoque1** [reply]
The [issue](https://app.zenhub.com/workspaces/alkemio-5ecb98b262ebd9f4aec4194c/issues/alkem-io/client-web/931).

**Reply by FloriSpoelstra** [reply]
I am pretty sure that I wasn't already a member of those ecoverses. So it does appear that a pending application is also published under "current memberships".

In that case there are two issues with memberships:
- Apply button doesn't disappear after you have become a member of a community.
- Pending membership is wrongfully published in your profile as membership (even before it has been accepted).

**Reply by Comoque1** [reply]
Just double checked - user wasn't part of any community and after applying to an Ecoverse, on the user profile is displayed only the pending application.
![image](https://user-images.githubusercontent.com/23620658/128858924-53e5f26d-81ec-4248-a80a-1193428d467d.png)

**Reply by FloriSpoelstra** [reply]
Ok, thanks for checking!

**Reply by Comoque1** [reply]
You are welcome!
